src/app/shared-services/db/migrations.ts

```diff
@@ -110,6 +110,7 @@ export const MIGRATIONS = [
     lon FLOAT NOT NULL,
     starttime DATETIME NOT NULL,
     endtime DATETIME NOT NULL,
+    observationcount FLOAT NOT NULL,
```

Comment:
Caution when changing table-definitions in the database: you are retrospectively changing a migration-step, that has already been merged to the `develop`-branch. This leads to inconsistencies: an update from the current state of the `develop`-branch to this version wouldn't include the change made in line 113, since the migration (lines 107ff. `add staypoint persistence`) already happened and won't be done again.

In this case it should be alright, because we did not publish said state to our testers – however we potentially need to reinstall the app (at least for me this is the case).

Reply:
Sidenote: this should be camelCase `observationCount`

Reply:
that makes a lot of sense, thanks for the detailed explanation. what would be your suggestion for moving ahead with this? 

Maybe as background, I included `observationcount` to spot periods of inactivity where the tracking has been turned off (by a user or by the OS) and is after a longer period of time turned on again, leading to a long staypoint covering the period of inactivity where the user could have moved around a bunch. Such long staypoints would mess with the inference of home/work or other POIs, thus having a low ratio of duration/observationcount can be used to filter these out. This workaround will of course be obsolete when we save start/stop times of recordings (see #15).

Reply:
We could either ignore it, since that version was not published (as mentioned above) or we add another migration step where the `observationcount` column is added. The latter approach might potentially also need to take care for the case when we already have some staypoints without `observationcount`. What do you think, @schrooom?

Reply:
Generally I think it is alright for the above stated reasons, but given the following statement, it might make sense to reevaluate. 

> This workaround will of course be obsolete when we save start/stop times of recordings (see #15).

When saving start/stop times of recordings is the way we're aiming for in the longterm plus the `observationcount` is more of a workaround, then it makes sense to me to simply implement persistence of the start/stop times in the first place, wouldn't it @jbraese + @zven?